src/k8s/cmd/k8s/k8s_bootstrap.go

```diff
@@ -40,10 +40,11 @@ func newBootstrapCmd(env cmdutil.ExecutionEnvironment) *cobra.Command {
 		address     string
 	}
 	cmd := &cobra.Command{
-		Use:    "bootstrap",
-		Short:  "Bootstrap a new Kubernetes cluster",
-		Long:   "Generate certificates, configure service arguments and start the Kubernetes services.",
-		PreRun: chainPreRunHooks(hookRequireRoot(env)),
+		Use:     "bootstrap",
+		GroupID: "cluster",
```

Comment:
why do you need to add the groupID here? 
This will be added via `addCommands`, no?

Reply:
Nevermind - updated already
(Great simplification btw)